.travis.yml

```diff
@@ -116,7 +116,7 @@ script:
   - flake8 --builtins=ArgumentError .
   # Run test with py.tests
   - if [[ $INSTALL_TYPE == 'pip_setup' ]]; then python setup.py test; fi
-  - if [[ $INSTALL_TYPE == 'conda' ]]; then py.test --durations=20 --maxfail=5 $COV devito tests/; fi
+  - if [[ $INSTALL_TYPE == 'conda' ]]; then py.test $COV devito tests/; fi
```

Comment:
why? I actually like those two options

Reply:
OK -- seen the change to .cfg, ignore this !

Reply:
You'll notice that adding it to .cfg also now prints the slowest cells for the notebooks testing which is nice.